### DISCUSSION TITLE
Проблема с обновлением списка, почему не работают мои прокси?

### DISCUSSION THREAD

**Discussion by silverjohnes** [anticensority/runet-censorship-bypass#336] [General]
Добрый день! В последние несколько дней расширение сменило синий цвет на серый. Судя по новостям, причина в "Сбое 07.12.24". Сбой выражается в ошибке Failed to load resource: the server responded with a status of 403 и невозможности загрузить обновленного скрипта. Почему из-за невозможности загрузить этот скрипт не работают мои собственные прокси и сайты, настроенные на работу через них?

**Comment by ilyaigpetrov** [comment]
У вас, видимо, выбран PAC-скрипт "АнтиЗапрет" вместо "Only own sites and only own proxies / Только свои сайты и свои прокси". В таком режиме выбранный PAC-скрипт (АнтиЗапрет) __дополняется__ вашими исключениями и прокси-серверами. Если этот базовый скрипт недоступен, то и исключения с серверами в него не пропишутся.

**Reply by silverjohnes** [reply]
Спасибо, буду знать. А почему, если новый скрипт не доступен, не работает старый, загруженный ранее?

**Reply by ilyaigpetrov** [reply]
Это ошибка в написанном мной коде. Подробнее пока не знаю.

**Reply by silverjohnes** [reply]
Благодарю за ответ и за отличное полезнейшее расширение!

**Reply by ilyaigpetrov** [reply]
@Olezha180, ваше сообщение было удалено как несоответсвующее теме данного обсуждения. Найдите более подходящую тему или создайте новую.
